README.md

```diff
@@ -1,32 +1,84 @@
-# ISLANDORA IP EMBARGO
-
-## BUILD STATUS
-Current build status:
-[![Build Status](https://travis-ci.org/Islandora/islandora_ip_embargo.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_ip_embargo)
-
+# Islandora IP Embargo[![Build Status](https://travis-ci.org/Islandora/islandora_ip_embargo.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_ip_embargo)
 CI Server:
 http://jenkins.discoverygarden.ca
 
-## CONTENTS OF THIS FILE
-
- * summary
- * configuration
-
-## SUMMARY
-
 A Drupal based approach to embargoing content based on IP ranges.
 This module is not integrated with the Islandora Embargo module.
 This module breaks islandora_xml_site_map with its redirects. 
 
-## CONFIGURATION
+## Requirements
+
+The following Drupal modules are required:
+
+* [Islandora](https://github.com/islandora/islandora)
+* [Rules](https://drupal.org/project/rules) (While not a dependency, more functionality can be found by installing this module.)
+
+## Installation
+
+Install as usual, see [this](https://drupal.org/documentation/install/modules-themes/modules-7) for further information.
+
+## Configuration
+
+### Drupal cron configuration
+
+Drupal cron should be configured to run once a day to clean out expired embargoes.
+
+### Network address lists
+
+Go to `yoursite.com/admin/islandora/tools/ip_embargo` to manage network address lists. You need at least one list to use IP embargo, because these lists are what will appear in the 'IP address range list' drop-down in the 'IP Embargo' tab for any given object, and you can't set an IP embargo without an IP address range list.
```

Comment:
Should avoid "you" in READMEs; language like "at least one list is required to use IP embargo" is preferred